delta/plugins/blazegraph/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/slowqueries/BlazegraphSlowQueryStoreSuite.scala

```diff
@@ -31,7 +32,7 @@ class BlazegraphSlowQueryStoreSuite
       SparqlQuery(""),
       failed = true,
       1.second,
-      Instant.now(),
+      Instant.now().truncatedTo(ChronoUnit.MILLIS),
```

Comment:
> [The clock used by `Instant.now()`] is based on the best available system clock. This may use [System.currentTimeMillis()](https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#currentTimeMillis--), or a higher resolution clock if one is available.

Reply:
I think in production code we are safe, as we should never be calling `Instant.now()` directly. In tests it's more annoying to have to use a clock.

One possible solution is to be a bit more relaxed in tests. I know that we use a fixed clock and check exact timestamps, but an alternative approach is to use a normal clock be a little more flexible. That is probably too big a change though

Perhaps there is some sort of typeclass equality that could rescue us..?